Headers/AppLovinSDK.framework/Headers/ALVariableService.h

```diff
@@ -0,0 +1,92 @@
+//
+//  ALVariableService.h
+//  AppLovinSDK
+//
+//  Created by Thomas So on 10/7/18.
+//  Copyright © 2020 AppLovin Corporation. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class ALVariableService;
+
+/**
+ * Use this service to retrieve values of variables that were pre-defined on AppLovin's dashboard.
+ */
+@interface ALVariableService : NSObject
+
+/**
+ * Returns the boolean value of the variable named by the given key.
+ * Returns `false` if no mapping of the desired type exists for the given key.
+ *
+ * @param key The name of the variable for which you want to retrieve the value.
+ *
+ * @return The value of the variable named by the value of `key`, or `nil` if no value was found.
+ */
+ // [PLP]
+ // - "no mapping of the desired type" -- who desires the type? how do they express this desire?
```

Comment:
These values are set in the UI (e.g. the pub can set the key string `"foo"` to values `true` or `"bar"` - the former value is a boolean, latter is a string).